### ISSUE TITLE
Cannot access protected property Joomla\CMS\Menu\MenuItem::$params

### ISSUE THREAD

**Issue by n3t** [templaza/astroid-framework#272]
Astroid Framework 2.6.3, Joomla 4, Menu Item Featured Articles, 
Call Stack
```
--
# | Function | Location
1 | () | JROOT/libraries/astroid/framework/template.php:78
2 | AstroidFrameworkTemplate->addMeta() | JROOT/libraries/astroid/framework/template.php:54
3 | AstroidFrameworkTemplate->__construct() | JROOT/templates/tz_pethub/html/com_content/featured/default_item.php:12
4 | include() | JROOT/libraries/src/MVC/View/HtmlView.php:410
5 | Joomla\CMS\MVC\View\HtmlView->loadTemplate() | JROOT/templates/tz_pethub/html/com_content/featured/default.php:39
6 | include() | JROOT/libraries/src/MVC/View/HtmlView.php:410
7 | Joomla\CMS\MVC\View\HtmlView->loadTemplate() | JROOT/libraries/src/MVC/View/HtmlView.php:203
8 | Joomla\CMS\MVC\View\HtmlView->display() | JROOT/components/com_content/src/View/Featured/HtmlView.php:207
9 | Joomla\Component\Content\Site\View\Featured\HtmlView->display() | JROOT/libraries/src/Cache/Controller/ViewController.php:102
10 | Joomla\CMS\Cache\Controller\ViewController->get() | JROOT/libraries/src/MVC/Controller/BaseController.php:634
11 | Joomla\CMS\MVC\Controller\BaseController->display() | JROOT/components/com_content/src/Controller/DisplayController.php:121
12 | Joomla\Component\Content\Site\Controller\DisplayController->display() | JROOT/libraries/src/MVC/Controller/BaseController.php:672
13 | Joomla\CMS\MVC\Controller\BaseController->execute() | JROOT/libraries/src/Dispatcher/ComponentDispatcher.php:143
14 | Joomla\CMS\Dispatcher\ComponentDispatcher->dispatch() | JROOT/components/com_content/src/Dispatcher/Dispatcher.php:56
15 | Joomla\Component\Content\Site\Dispatcher\Dispatcher->dispatch() | JROOT/libraries/src/Component/ComponentHelper.php:355
16 | Joomla\CMS\Component\ComponentHelper::renderComponent() | JROOT/libraries/src/Application/SiteApplication.php:200
17 | Joomla\CMS\Application\SiteApplication->dispatch() | JROOT/libraries/src/Application/SiteApplication.php:241
18 | Joomla\CMS\Application\SiteApplication->doExecute() | JROOT/libraries/src/Application/CMSApplication.php:294
19 | Joomla\CMS\Application\CMSApplication->execute() | JROOT/includes/app.php:61
20 | require_once() | JROOT/index.php:32

```

IMHO in framework/template.php on line 78 should be `$item->getParams()` instead of `$item->params` for Joomla 4

**Comment by sonvnn** [comment]
Hi @n3t 

Thanks for report bug. This is problem of PetHub template. I will soon update PetHub with latest Astroid 2.6.3

Best Regards,
Sonny

**Comment by sonvnn** [comment]
I fixed this issue in PetHub v1.0.6. Please help me update your PetHub template. I will close this issue as resolved.

Thanks & Best Regards,
Sonny
